crates/freeze/src/datasets/traces.rs

```diff
@@ -62,6 +68,42 @@ impl CollectByTransaction for Traces {
         process_traces(&traces, columns, &query.schemas)
     }
 }
+
+pub(crate) fn filter_traces_by_from_to_addresses(
+    traces: Vec<Trace>,
+    from_address: &Option<Vec<u8>>,
+    to_address: &Option<Vec<u8>>,
+) -> Vec<Trace> {
+    // filter by from_address
+    let from_filter: Box<dyn Fn(&Trace) -> bool + Send> = if let Some(from_address) = from_address {
+        Box::new(move |trace| {
+            let from = match &trace.action {
+                Action::Call(action) => action.from,
+                Action::Create(action) => action.from,
+                Action::Suicide(action) => action.address,
+                Action::Reward(action) => action.author,
```

Comment:
`Action::Reward(action) => action.author,` should be to in the to filter not the from filter

Reply:
resolved in [9bad7a5](https://github.com/paradigmxyz/cryo/pull/123/commits/9bad7a5af9e10e26723b5a622a5eb9d6b14432a7)